example/cumulus-tf/python_reference_workflow.tf

```diff
@@ -0,0 +1,112 @@
+
+resource "aws_sfn_activity" "ecs_task_python_processing_service" {
+  name = "${var.prefix}-EcsTaskPythonProcess"
+  tags = local.tags
+}
+
+module "python_processing_service" {
+  source = "../../tf-modules/cumulus_ecs_service"
+
+  prefix = var.prefix
+  name   = "PythonProcess"
+  tags   = local.tags
+
+  cluster_arn                           = module.cumulus.ecs_cluster_arn
+  desired_count                         = 1
+  image                                 = "cumuluss/cumulus-process-activity:1"
```

Comment:
So is this what we talked about getting re-built with every release?

Reply:
No - because of the way cumulus-process-py is utilized, *users* won't be using this image.  As such the release process probably shouldn't be tied to Core release versions (similar to the CMA image configuration/release process)